client/src/app/components/Autocomplete.tsx

```diff
@@ -266,12 +266,40 @@ export const Autocomplete: React.FC<IAutocompleteProps> = ({
   const menu = (
     <Menu
       ref={menuRef}
-      onSelect={onSelect}
-      onKeyDown={handleMenuKeyDown}
+      // onSelect={handleMenuOnSelect}
```

Comment:
Seems to be handled at the menuItem now which I understand is why this is commented out. Cleanup/remove?

Reply:
I forgot to take it out.  Good eye!